composer.json

```diff
@@ -29,14 +30,14 @@
         "twig/twig": "~1.12|~2.0",
         "doctrine/orm": "~2.1",
         "doctrine/phpcr-odm": "^1.3",
-        "jackalope/jackalope-doctrine-dbal": "^1.0",
+        "jackalope/jackalope-doctrine-dbal": "^1.1.5",
         "propel/propel1": "~1.7",
-        "symfony/yaml": "2.*",
-        "symfony/translation": "~2.0",
+        "symfony/yaml": "^2.0.5",
+        "symfony/translation": "^2.1",
         "symfony/validator": "^2.2",
         "symfony/form": "~2.1",
-        "symfony/filesystem": "2.*",
-        "phpunit/phpunit": "^4.0|^5.0"
+        "symfony/filesystem": "^2.1",
+        "phpunit/phpunit": "^5.0"
```

Comment:
you also need to allow phpunit 4.8, as PHPUnit 5 is not compatible with PHP 5.5 (or you should rely on the phar available on Travis)